cmd/find-main.go

```diff
@@ -85,6 +85,10 @@ var (
 			Name:  "watch",
 			Usage: "monitor a specified path for newly created object(s)",
 		},
+		cli.BoolFlag{
+			Name:  "count",
+			Usage: "only print count numbers for matched object(s)",
```

Comment:
%s/only print count numbers for matched object(s)/only print numbers of matched object(s)/g